test

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -e
+
```

Comment:
Sure, but it adds about 2 seconds on my laptop to running the tests. Probably for the best, so I don't confuse myself in the future as to why changes to `lib/` aren't being reflected in the tests.